### PR TITLE
Update v1.4.0

### PR DESCRIPTION
**Pull request by rafaelmaeuer**
- chore: update express-openapi-validator to `v5`
- chore: update google-github-actions to `v1`

### CODE REVIEW

**Review by frytg — APPROVED** [review]

**Comment by rafaelmaeuer** [comment]
Find Cause for Test Errors:

- POST /events/de.ard.eventhub.v1.radio.track.playing (publish a new event)
- GET /topics (list all available topics)
  ```
  Uncaught AssertionError: expected 'text/plain; charset=utf-8' to include 'application/json'
  ```

- POST /subscriptions (test auth for POST /subscriptions)
  ```
  Uncaught AssertionError: expected { Object (_events, _eventsCount, ...) } to have status code 403 but got 400
  ```

- POST /subscriptions (add a new subscription to this user)
  ```
  Uncaught AssertionError: expected { Object (_events, _eventsCount, ...) } to have status code 201 but got 400
  ```

**Comment by rafaelmaeuer** [comment]
Failing with `Could not load the default credentials` which might correlate with missing `GOOGLE_APPLICATION_CREDENTIALS`: https://stackoverflow.com/a/47867652/9915210

Added env-checks accordingly, now failing with `GOOGLE_APPLICATION_CREDENTIALS not found`

File: config/index.js [hunk 1]
```diff
@@ -9,25 +9,30 @@
...
 // read env vars
 const stage = process.env.STAGE.toLowerCase()
 const port = process.env.PORT || 8080
 
+const exitWithError = (message) => {
+	logger.log({
+		level: 'error',
+		message,
+		source: 'config',
+		data: { env: process.env },
```
Comment (frytg) [inline config/index.js]: logging all envs here is a security risk since it can include sensitive details like `FIREBASE_API_KEY`

**Review by frytg — APPROVED** [review]

**Comment by rafaelmaeuer** [comment]
Was deployed to prod today

**Pull request merged**